flocker/acceptance/endtoend/test_dataset.py

```diff
@@ -10,6 +10,7 @@
 from twisted.trial.unittest import TestCase
 
 from ...common import loop_until
+from ...testtools import flaky
 
 from ..testtools import (
     require_cluster, require_moving_backend, create_dataset, DatasetBackend
```

Comment:
okay, if you're happy reading that everyday and don't get confused than I am happy too.
my untrainned eyes to flocker code combined with a small laptop screen raised some doubts